app/src/main/java/ca/unb/mobiledev/reflexrevolution/utils/Instruction.java

```diff
@@ -3,6 +3,9 @@
 public enum Instruction {
     BUTTON, //Temp instruction for testing
     TAP,
+    DOUBLETAP,
+    HOLD,
+    DONTTAP,
```

Comment:
I suggest we rename the instructions as `DOUBLE_TAP` and `DONT_TAP` to make them more readable.

Reply:
Good idea, I'll change that